### ISSUE TITLE
Trouble getting it to work with hasMany and belongsTo both in the same model

### ISSUE THREAD

**Issue by v-ken** [frank06/ember-data-save-relationships#2]
Having both hasMany and belongsTo in the same model causes this error to occur:

`Uncaught Error: Model '<app@model:recipe::ember765:null>' has no belongsTo relationship named 'ingredients' defined.`

Model:

```
export default DS.Model.extend({
  title: DS.attr(),

  // Relationships
  ingredients: DS.hasMany('ingredient'),
  author: DS.belongsTo('user')
});
```

When I remove the `author` from the model, everything works fine.

**Comment by frank06** [comment]
Should be fixed, can you try with `0.0.4`?

**Comment by v-ken** [comment]
Thanks Frank! Works like a charm!
